pkg/networkservice/metrics/server.go

```diff
@@ -44,10 +45,12 @@ func (s *metricsServer) Request(ctx context.Context, request *networkservice.Net
 	if conf == nil {
 		return nil, errors.New("VPPAgent config is missing")
 	}
+
 	ifaces := conf.GetVppConfig().GetInterfaces()
 	if len(ifaces) == 0 {
-		return nil, errors.New("VPPAgent config should contain at least one interface")
+		return next.Server(ctx).Request(ctx, request)
```

Comment:
Let's add logging here to notify that the metrics server is not used.

Reply:
Added.